docs/_documentations/container-registry-credentials.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: docs
+title: Container registry credentials for Codewind
```

Comment:
We had a discussion with design last week and decided to call them "Image Registries" rather than "Container Registries"